src/prefect/agent/local/agent.py

```diff
@@ -136,29 +138,37 @@ def deploy_flow(self, flow_run: GraphQLResult) -> str:
         """
         self.logger.info("Deploying flow run {}".format(flow_run.id))  # type: ignore
 
-        if not isinstance(
-            StorageSchema().load(flow_run.flow.storage),
-            (Local, Azure, GCS, S3, GitHub, GitLab, Webhook),
-        ):
+        storage = StorageSchema().load(flow_run.flow.storage)
+        if isinstance(storage, Docker):
             self.logger.error(
-                "Storage for flow run {} is not a supported type.".format(flow_run.id)
+                "Flow run %s has an unsupported storage type: `%s`",
+                flow_run.id,
+                type(storage).__name__,
             )
-            raise ValueError("Unsupported Storage type")
-
-        env_vars = self.populate_env_vars(flow_run=flow_run)
-        current_env = os.environ.copy()
-        current_env.update(env_vars)
-
-        python_path = []
-        if current_env.get("PYTHONPATH"):
-            python_path.append(current_env.get("PYTHONPATH"))
-
-        python_path.append(os.getcwd())
-
-        if self.import_paths:
-            python_path += self.import_paths
-
-        current_env["PYTHONPATH"] = ":".join(python_path)
+            raise TypeError("Unsupported Storage type: %s" % type(storage).__name__)
+
+        # If the flow is using a run_config, load it
+        if getattr(flow_run.flow, "run_config", None) is not None:
```

Comment:
(rather than a truthy `if foo: ...`)

Reply:
Just a preference. I find it clearer to only check for `None`, otherwise I have to think about if any of the values that could be set on `.run_config` support truthiness. This way it's clear on inspection - it's either `None` or it's a `RunConfig` object.